js/content/community.js

```diff
@@ -291,6 +292,260 @@ let CommentHandler = (function(){
         observer.observe(document.body, { childList: true });
     };
 
+    function loadScripts() {
+        return new Promise(async function(res) {
+            scriptsLoaded = true;
```

Comment:
`scriptsLoaded` (past tense) is true before the scripts are loaded?

Reply:
Yes, it's a race condition basically.

Reply:
I'm commenting on the English. `scriptsLoaded = true` implies "The scripts are loaded," when it's set when the scripts *start* loading. It's really trivial...

Reply:
Okay